addons/fire/compat_medical_engine/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -10,6 +10,7 @@ class EGVAR(medical_treatment,actions) {
         allowSelfTreatment = 1;
         medicRequired = 0;
         treatmentTime = 5;
+        treatmentTimeTrained = 5;
```

Comment:
```suggestion
```
If they are the same, then I think it's best to remove
In case another mod uses inheritance to add actions
```
myTreatment: ace_BasicBandage {
treatmentTime = x;
```